kong/content.md

```diff
@@ -0,0 +1,56 @@
+# What is Kong?
+
+Kong was built to secure, manage and extend Microservices & APIs. If you're building for web, mobile or IoT (Internet of Things) you will likely end up needing to implement common functionality on top of your actual software. Kong can help by acting as a gateway for any HTTP resource while providing logging, authentication and other functionality through plugins.
+
+Powered by NGINX and Cassandra with a focus on high performance and reliability, Kong runs in production at Mashape where it has handled billions of API requests for over ten thousand APIs.
+
+Kong's documentation can be found at [getkong.org/docs](http://getkong.org/docs).
+
+%%LOGO%%
+
+# How to use this image
+
+First, Kong requires a running Cassandra cluster before it starts. You can either use the [mashape/cassandra](https://github.com/Mashape/docker-cassandra) image, provision a test instance on [kongdb.org](http://kongdb.org) or use a cluster of your own.
+
+## 1. Link Kong to a Cassandra container
+
+Start a Cassandra container by doing so:
+
+```shell
+$ docker run -d -p 9042:9042 --name cassandra mashape/cassandra
```

Comment:
The `mashape/cassandra` image should be deprecated (ideally, removed) and the example should use the official `cassandra` image (which did not exist at that time), with the `2.2` tag.